rules/falco_rules.yaml

```diff
@@ -2997,7 +2997,7 @@
      image=%container.image.repository namespace=%k8s.ns.name
      fd.rip.name=%fd.rip.name fd.lip.name=%fd.lip.name fd.cip.name=%fd.cip.name fd.sip.name=%fd.sip.name)
   priority: WARNING
-  tags: [maturity_sandbox, container, network, mitre_discovery, T1046]
+  tags: [maturity_incubating, container, network, mitre_discovery, T1046]
```

Comment:
"Network Connection outside Local Subnet" was tagged as one of the initially relevant PCI DSS rule in https://falco.org/blog/falco-pci-controls/.